### DISCUSSION TITLE
loading saved data only works with intpath but not dpath

### DISCUSSION THREAD

**Discussion by tom221085** [miniscope/minian#146] [Q&A] [answered]
Hi,

As mentioned in the title, when I try to load already processed data to save it into a different format using 'minian_ds = open_minian()' it only works with intpath but not dpath. I get the following error message if I use dpath:

![Web capture_16-7-2021_151819_localhost](https://user-images.githubusercontent.com/11939642/125998172-ae551104-73f6-47d4-9da2-ad5c8c234168.jpeg)

I found that there are more folders in intpath than dpath. Is there anything I have to save additionally in the dpath/minian folder before I can reload data to save it in a different format?

Thanks,
Thomas

**Comment by phildong** [comment, marked as answer]
Hey your snapshot shows an exception when you run `open_minian(intpath)`, is that the problem? Could you send a snapshot of the folder where `intpath` is pointing to (or whichever path that's giving you the problem)?

**Reply by tom221085** [reply]
Hi phildong,

I think the problem is, that I directly access the data on our shared lab storage and for some reason that causes troubles. I just copied it on my computer in the minian_intermediate folder and now it works. Thanks!
